### PR TITLE
Don't allow font generation without fontforge

### PR DESCRIPTION
**Pull request by buoyad**
Before, it would still write the font, but the metrics would be wrong. r? @keybase/react-hackers

### CODE REVIEW

**Comment by adamjspooner** [comment]
Does this fix the problem we were seeing with @zanderz's PR?

File: shared/desktop/yarn-helper/font.js [hunk 1]
```diff
@@ -77,6 +77,18 @@ const getSvgPaths = skipUnmatchedFile =>
  * For config options: https://github.com/sunflowerdeath/webfonts-generator
  */
 function updateIconFont(web) {
+  if (!web) {
+    // Check if fontforge is installed, required to generate the font
+    try {
+      execSync('fontforge')
+    } catch (e) {
+      if (e.message.includes('not found')) {
+        throw new Error('FontForge is required to generate the icon font. Install and try again.')
```
Comment (chrisnojima) [inline shared/desktop/yarn-helper/font.js]: you should say they need to run `yarn` and not `yarn modules` and install fontforge globally

Comment (buoyad) [reply shared/desktop/yarn-helper/font.js]: fontforge isn't installed by yarn though

Comment (chrisnojima) [reply shared/desktop/yarn-helper/font.js]: yeah additionally they need to install fontforge globally, they need both and its not the usual flow

**Comment by buoyad** [comment]
@adamjspooner #16189 doesn't generate the font. Unless the font in master was generated without fontforge I don't think this fixes it.

**Pull request merged**